pkg/utils/utils_virtual.go

```diff
@@ -42,7 +42,7 @@ var (
 )
 
 const (
-	ospMetaDataDir = "/host/var/config/openstack/latest/"
+	ospMetaDataDir = "/host/var/config/openstack/2018-08-27"
```

Comment:
how often does this date change?

Reply:
> > Machines are immutable, so there is no need to fetch for metadata every 30s.
> 
> Is there a use case of hot plugging?

no, OpenStack doesn't support it. Nova will fail if we hotplug an SR-IOV device to a VM.

Reply:
this version is the latest tag, so we can safely use it for now and new versions don't come often.... last one was in 2018, so...